### DISCUSSION TITLE
Multidimensional Time Series Analysis with Continuous and Categorical Data

### DISCUSSION THREAD

**Discussion by Sandy4321** [TDAmeritrade/stumpy#38] [Q&A]
great code , thanks
can it be used for multidimensional time series with both continues and categorical data

**Comment by seanlaw** [comment]
It can only be used for multi-dimensional time series with floating point numbers. It is not obvious how one would use this for categorical time series data since the meaning behind the statistical calculations would not make any sense. Please see issue #23 for more discussions about this.

**Comment by seanlaw** [comment]
@Sandy4321 closing this for now but feel free to reopen if you have further questions.
